packages/babel-parser/test/helpers/runFixtureTests.js

```diff
@@ -1,3 +1,5 @@
+/* global BigInt */
```

Comment:
We'll be able to remove this when we enable the latest version of `@babel/eslint-config-internal`.